core/libcamera_app.hpp

```diff
@@ -184,6 +200,7 @@ class LibcameraApp
 	void stopPreview();
 	void previewThread();
 	void configureDenoise(const std::string &denoise_mode);
+	Mode selectModeForFramerate(const libcamera::Size size, double fps);
```

Comment:
I guess `&size` is actually no better than `size` for a small structure like this, is it?

Reply:
Shouldn't matter too much, but I'll switch to using a reference.